docs/swagger.yaml

```diff
@@ -835,7 +882,7 @@ components:
           type: string
           description: 'Can be either country code or country name. Example: "NO"'
       additionalProperties: false
-      description: Information about the customer to be prefilled
+      description: "Information about the customer to be prefilled.\r\n            \r\nIf any of the customer information is invalid such as the phone number,\r\nthe customer will be prompted to input new user information."
```

Comment:
Are these line breaks right?

Reply:
Yes, it looks okay in swagger at least. 
![image](https://user-images.githubusercontent.com/110486952/198583291-c20f1dd6-e902-4573-9def-eda14f6b6472.png)